Gemfile.lock

```diff
@@ -773,7 +773,7 @@ GEM
     webrick (1.6.0)
     websocket-driver (0.7.1)
       websocket-extensions (>= 0.1.0)
-    websocket-extensions (0.1.4)
+    websocket-extensions (0.1.5)
```

Comment:
Probably going to result in a conflict. I think you want to merge the latest master. See: https://github.com/department-of-veterans-affairs/vets-api/pull/4368/files

Reply:
Ok, done.